### DISCUSSION TITLE
influxdb2?

### DISCUSSION THREAD

**Discussion by HerrVonundZu** [breadlysm/SpeedFlux#7] [Q&A]
Hey there,
thx for the nice work.
i am trying to implement it on my syno nas with an InfluxDB2 instance.
is there any way to send the Data to one of the Buckets?
Have a nice day

**Comment by breadlysm** [comment]
Apologies, I did not see this :( 

I think Adding InfluxDB2 should be fairly easy. I'll make an issue to see about adding that functionality.

**Comment by elzik** [comment]
I'm not sure if there has been any progress with adding v2 support but it looks like InfluxDb v2 can support access from clients using a v1 API.

Using the details from [the InfluxDb community](https://community.influxdata.com/t/getting-started-with-influxdb-docker-401-unauthorized/16989/3) I managed to:
- Create a mapping from the v1 SpeedFlux database to a v2 bucket
- Create a legacy authentication for the user that SpeedFlux tries to use

However, I still get a `"Couldn't initiate InfluxDB <2", InfluxDBClientError('401: {"code":"unauthorized","message":"Unauthorized"}')` in the SpeedFlux log. Any pointers here would be appreciated.
